### ISSUE TITLE
Add support for a `runAction` in the `plop` API

### ISSUE THREAD

**Issue by crutchcorn** [plopjs/node-plop#169]
Currently, if I'm inside of a `plop action` and want to run a `add` or `addMany`, my option is really reduced to making a new generator and running the generator's `generator.runActions`.

It would be really nice if instead of this:

```javascript
const {runGeneratorActions} = require('node-plop/lib/generator-runner');

plop.setActionType('setupAzureNPM', async (answers, config, plop) => {
  await runGeneratorActions({
    name: 'Azure NPM Setup',
      actions: [
        {
          type: 'add',
          // ...
        }
      ]
  })
});
```

We could have something like this:

```javascript
plop.setActionType('setupAzureNPM', async (answers, config, plop) => {
  await plop.runAction({
    type: 'add',
    // ...
   })
});
```

To run a single action from the plop API

**Comment by amwmedia** [comment]
yeah, I would just look at PR #58. It was put up a long time ago and I always meant to merge it, but I wanted to see if I could get the full plop API injected into the function. Anyway, it might be worth just merging it as is.

**Comment by crutchcorn** [comment]
I think I agree with you, though. I'd rather inject the API into `plopAPI` itself. I have a branch where I'm working on just that. I'll see if I can clean it up this week

**Comment by crutchcorn** [comment]
Transfering this issue to https://github.com/plopjs/plop/ since we're migrating all of `node-plop` to that repo as a monorepo
